### DISCUSSION TITLE
Issue with Student Developer Pack - Unable to Access GitHub Pro Features

### DISCUSSION THREAD

**Discussion by liangDarwin2** [community/community#72691] [GitHub Education]
### Hi everyone,

Dear GitHub Support Team,

I hope this email finds you well. I'm reaching out to address a concern I have encountered after successfully registering for the GitHub Pro Student Developer Pack.

 On October 24, 2023, I received a confirmation that my application for the Student Developer Pack had been approved. However, since then, I've been unable to access the GitHub Pro features that are supposed to be available as part of the package.

Here are some details to assist with your troubleshooting:

GitHub Username: liangDarwin2

Github link: https://github.com/liangDarwin2

Date of Approval: October 24, 2023

I have double-checked to ensure that I'm logged in to the correct account and have tried logging out and back in, as well as attempting access from different devices, but the issue persists.

It would be greatly appreciated if you could look into this matter and provide guidance or a solution. I was eagerly looking forward to utilizing the advanced features of GitHub Pro to aid in my academic projects.

Thank you in advance for your assistance and understanding. I'm hopeful for a swift resolution so I can make the most of the resources provided by the Student Developer Pack.

Best regards,
Gaoquan Liang

**Comment by iprattham** [comment]
Check for account settings: While you wait for a response from GitHub Support, it's a good idea to review your account settings and preferences to make sure there are no restrictions or settings that might be causing the issue.

Follow up: If you don't receive a response within a reasonable timeframe, don't hesitate to follow up with GitHub Support via their official mail or twitter. Sometimes, support inquiries can be overlooked, and a gentle reminder can help expedite the resolution.
